Client/Ecosia/AppDelegate/EcosiaAppDelegate.swift

```diff
@@ -0,0 +1,74 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Shared
+
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+    var coordinator: MainCoordinator?
+    var orientationLock = UIInterfaceOrientationMask.all
+    var tabManager: TabManager!
```

Comment:
Is this actually is used? 

I see `browserViewController` is since the workaround is still in place, but I think the `tabManager` could even be removed since you've already refactored it into the factory and passed it to any delegate that needed it, isn't that true?

Reply:
`tabManager` could be removed as I made the factory method after that implementation. Forgot to remove it.